handbook/engineering/hiring/software-engineer-code-insights.md

```diff
@@ -0,0 +1,114 @@
+- Build accessible UIs that other engineers and engineering leaders love using.
+- Collaborate with backend engineers to define the ideal APIs needed to deliver on the desired user experience.
+- Collaborate with other product teams like search, code intelligence and campaigns to figure out integration points with other Sourcegraph features.
+
+## Qualifications
+
+- **Required**:
+  - Experience in writing [SPAs](https://en.wikipedia.org/wiki/Single-page_application) using React or similar component-based frontend frameworks.
+  - Familiarity with TypeScript or a different typed programming language.
+  - Skilled at writing clean, reusable CSS and semantic HTML to implement visual designs.
+  - Experience in designing backend APIs (e.g. GraphQL or REST) from a client perspective.
+  - Experience in using backend APIs from the frontend.
+  - Skilled at building and testing (e.g. unit testing, automated end-to-end testing) UIs.
+- **Nice-to-haves**:
+  - Extensive experience with TypeScript and/or GraphQL.
+  - Contributions to open source projects.
+  - Experience working on small high-performing product teams.
+  - Experience working with a global or otherwise multicultural team.
+  - Developer platform/tool industry experience.
+  - Experience in the space of data visualization dashboards or similar.
+  - Interest in measuring code and engineering best practices.
+
+## Not sure if this is you?
+
+We want a diverse, global team, with a broad range of experience and perspectives. If this job sounds great, but you're not sure if you qualify, apply! Candidates often underestimate how qualified they actually are. We carefully consider every application, and will either move forward with you, find another team that might be a better fit, keep in touch for future opportunities, or thank you for your time.
+
+## Learn more about us
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](https://about.sourcegraph.com/company/remote) that is [distributed across the world](https://about.sourcegraph.com/company/team). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](https://about.sourcegraph.com/handbook/people-ops/compensation) and [practical benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](https://about.sourcegraph.com/handbook).
+
+## Interview process
+
+<!-- 1. You [apply here](https://jobs.lever.co/sourcegraph/73fda68b-c821-4627-af07-41a0850072fb/apply). -->
+1. The **Hiring Manager** reviews all the information you provided on your application to determine if you meet our qualifications for this role (if there is another open role we think you would be better qualified for, we will let you know).
```

Comment:
Who is the HM for this role?

Reply:
Me. Should that be called out in the handbook? I copied this from the frontend JD

Reply:
Yes, I think candidates want to know who the HM is (we should add it to other JDs that have it missing). Also, since you are the HM here, I would want Jean to be in one of the interviews (perhaps instead of Loïc)